src/hooks/core/chat/useFetchChannelHook.ts

```diff
@@ -103,7 +103,6 @@ const useFetchChannelHook = () => {
 
       await Promise.all(
         (channel?.messages || []).map(async (message) => {
-          if (message?.type === 'deleted') return;
           const chat = ChatSchema.fromGetAllChannelAPI(channel?.id, message);
           await chat.save(localDb);
         })
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change removes the condition that skips saving messages of type 'deleted'. If this was intentional, it's fine. However, if you still want to skip 'deleted' messages, you should reintroduce the condition.
```diff
- if (message?.type === 'deleted') return;
  const chat = ChatSchema.fromGetAllChannelAPI(channel?.id, message);
  await chat.save(localDb);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->